runtime/src/main/java/com/datastax/oss/quarkus/runtime/graal/NativeSubstitutions.java

```diff
@@ -15,20 +15,40 @@
  */
 package com.datastax.oss.quarkus.runtime.graal;
 
+import com.datastax.oss.driver.internal.core.os.CpuInfo;
 import com.datastax.oss.driver.internal.core.os.Native;
+import com.oracle.svm.core.annotate.KeepOriginal;
 import com.oracle.svm.core.annotate.Substitute;
 import com.oracle.svm.core.annotate.TargetClass;
 
 @TargetClass(Native.class)
+@Substitute
 final class NativeSubstitutions {
 
-  /**
-   * This method returns always false because jnr is not available.
-   *
-   * @return false denoting that {@link Native#getProcessId()} cannot be called.
-   */
+  @Substitute private static final CpuInfo.Cpu CPU = CpuInfo.determineCpu();
+
   @Substitute
   public static boolean isGetProcessIdAvailable() {
     return false;
   }
+
+  @Substitute
+  public static int getProcessId() {
+    throw new RuntimeException("Native operations are not supported on this platform");
```

Comment:
`UnsupportedOperationException` would be slightly better, but I guess this doesn't matter, the method is never going to be called anyway, right?

Reply:
That's true at the moment @adutra , but we're relying on the user to check whether it's available before they make the call... so it's certainly something that could happen.

I agree that UOE is more directly applicable here, but it's worth noting that the "real" Native throws an IllegalStateException in this case (see https://github.com/datastax/java-driver/blob/f4ea838b7650ab5bb10d089e1ea8613ba2592b05/core/src/main/java/com/datastax/oss/driver/internal/core/os/Native.java#L74)... so I'm inclined to mirror that.